dask/async.py

```diff
@@ -414,29 +429,27 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
 
     state = start_state_from_dask(dsk, cache=cache)
 
-    tick = [0]
-
     if state['waiting'] and not state['ready']:
         raise ValueError("Found no accessible jobs in dask")
 
+    if not execute_cm:
+        execute_cm = default_cm
```

Comment:
It could, I just thought `None` made more sense to be user facing than an opaque function. A default would be fine too.